2002-08-23  Jakub Jelinek  <jakub@redhat.com>

	* config/alpha/alpha.c (alpha_encode_section_info): Use
	MODULE_LOCAL_P.

--- gcc/config/alpha/alpha.c.jj	2002-05-30 11:12:20.000000000 +0200
+++ gcc/config/alpha/alpha.c	2002-08-23 12:05:30.000000000 +0200
@@ -1594,7 +1594,7 @@ alpha_encode_section_info (decl)
   /* Linkonce and weak data is never local.  */
   else if (DECL_ONE_ONLY (decl) || DECL_WEAK (decl))
     is_local = false;
-  else if (! TREE_PUBLIC (decl))
+  else if (! TREE_PUBLIC (decl) || MODULE_LOCAL_P (decl))
     is_local = true;
   /* If PIC, then assume that any global name can be overridden by
      symbols resolved from other modules.  */
